package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "csv-split-stream",
+  "version": "1.0.0",
+  "description": "Splits a CSV read stream into multiple write streams by number of lines",
```

Comment:
I would remove "by number of lines". It is implied that it will be split in some way.